### DISCUSSION TITLE
Multiplicity: Spin vs UHF Keywords

### DISCUSSION THREAD

**Discussion by sbembenek18** [grimme-lab/xtb#1045] [Q&A] [answered]
I have a system with 2 unpaired electrons -- Multiplicity 3. Using --uhf 2 seems to handle this correctly, but when I use the $spin 1 in the input file, I get a different result. Are these keywords describing two different things?

**Comment by marcelmbn** [comment, marked as answer]
I'd have to look it up myself but I guess, for reaching the equivalent result, you'd have to set `$spin 2`.
